.examples/README.md

```diff
@@ -0,0 +1,60 @@
+# Examples section
+
+In this subfolders are some examples how to use the docker image. There are two sections:
+ 
+ * [`dockerfiles`](https://github.com/nextcloud/docker/tree/master/.examples/dockerfiles)
+ * [`docker-compose`](https://github.com/nextcloud/docker/tree/master/.examples/docker-compose)
+
+The `dockerfiles` are derived images, that add or alter certain functionalities of the default docker images. In the `docker-compose` subfolder are examples for deployment of the application, including database, redis, collabora and other services.
+
+## Dockerfiles
+The Dockerfiles use the default images as base image and build on top of it.
+
+
+Example | Description
+------- | -------
+[cron](https://github.com/nextcloud/docker/tree/master/.examples/dockerfiles/cron) | uses supervisor to run the cron job inside the container (so no extra container is needed).
+[imap](https://github.com/nextcloud/docker/tree/master/.examples/dockerfiles/imap) | adds dependencies required to authenticate users via imap
+[smb](https://github.com/nextcloud/docker/tree/master/.examples/dockerfiles/smb) | adds dependencies required to use smb shares
+
+
+
+
+
+## docker-compose
+In `docker-compose` additional services are bundled to create a complete nextcloud installation. The examples are designed to run out-of-the-box.
+Before running the examples you have to modify the `db.env` and `docker-compose.yml` file and fill in your custom information.
+
+The docker-compose examples make heavily use of dereived Dockerfiles to add configuration files into the containers. This way they should also work on remote docker systems as _Docker for Windows_. When running docker-compose on the same host as the docker daemon, another possibility would be to simply mount the files in the volumes section in the `docker-compose.yml` file.
+
+
+### insecure
+
+To use this example complete the following steps:
```

Comment:
add a sentence why this setup is insecure and that it should not be used in production.

Reply:
done